pkg/util/xds/stats_callbacks.go

```diff
@@ -114,17 +134,30 @@ func (s *statsCallbacks) OnStreamOpen(context.Context, int64, string) error {
 	return nil
 }
 
-func (s *statsCallbacks) OnStreamClosed(int64) {
+func (s *statsCallbacks) OnStreamClosed(streamID int64) {
```

Comment:
Do you know if these OnClosed methods are called during the graceful CP shutdown? It'd be not ideal to have stale metrics after each CP rollout

Reply:
What do you mean? Even if OnStreamClosed is not closed, the CP goes down, so it won't report this metric.